AutoCollection/Statsbeat.ts

```diff
@@ -70,18 +73,31 @@ class Statsbeat {
                     time: +new Date
                 };
                 this._handle = setInterval(() => {
-                    this.trackStatsbeatMetrics().catch((error) => {
+                    this.trackShortIntervalStatsbeats().catch((error) => {
                         // Failed to send Statsbeat
                         Logging.info(Statsbeat.TAG, error);
                     });
                 }, Statsbeat.NETWORK_STATS_COLLECTION_INTERVAL);
```

Comment:
can you rename this constant to be short_interval similar to 'LONG_STATS_COLLECTION_INTERVAL'?